docs/user/openstack/install_upi.md

```diff
@@ -854,7 +856,9 @@ $ oc get pods -A
 $ ansible-playbook -i inventory.yaml down-bootstrap.yaml
 ```
 
-The teardown playbook deletes the bootstrap port, server and floating IP address.
+The teardown playbook deletes the bootstrap port and server.
```

Comment:
Will we have problems in [here](https://github.com/openshift/installer/blob/master/upi/openstack/down-bootstrap.yaml#L16) if no fip is connected to the bootstrap machine?

Reply:
True! We don't create, so we shouldn't delete. Thanks!